compiler/IREmitter/Payload/codegen-payload.c

```diff
@@ -2315,49 +2315,60 @@ void sorbet_throwReturn(VALUE retval) {
     sorbet_ec_jump_tag(ec, ec->tag->state);
 }
 
-// This is invoked at the beginning of a method's body, and is analogous to EC_PUSH_TAG followed by the setjmp in
-// EC_EXEC_TAG:
+// This is invoked at the beginning of a method's body, and is analogous to EC_PUSH_TAG + EC_EXEC_TAG
```

Comment:
Would you mind extending this comment to say that it's safe to return after the call to setjmp because this function will always be inlined?

Reply:
I can, but I confess I don't understand the rationale for doing so.  Why wouldn't it be safe to return after a `setjmp`?  (And if it is unsafe to return after a `setjmp`, why wouldn't that same rationale apply to any function that calls `setjmp` anywhere else in all of `libruby.so`?)

Reply:
Oh, I see, the `setjmp_buf` is really only valid for the lifetime of the function in which you call `setjmp`.  OK, that makes sense, I can add a comment for that.